st2client/st2client/utils/httpclient.py

```diff
@@ -29,7 +29,7 @@ def add_ssl_verify_to_kwargs(func):
     def decorate(*args, **kwargs):
         if isinstance(args[0], HTTPClient) and 'https' in getattr(args[0], 'root', ''):
             cacert = getattr(args[0], 'cacert', None)
-            kwargs['verify'] = cacert if cacert else False
+            kwargs['verify'] = cacert if cacert else True
```

Comment:
It defaults to `True` since it would check for `certifi` CAs—including our self-signed one, which we're supposed to push to the certifi bundle. 

On a different note, we should really resurrect our efforts to get rid of the "insecure request" warning.